### ISSUE TITLE
Weird speed performance on CPU and worse accuracy than your provided weights.

### ISSUE THREAD

**Issue by nulli9** [lizhengwei1992/mobile_phone_human_matting#2]
Thank your for sharing the awesome project.

I tried to train the model on supervisely dataset by following the steps:
1. Resize downloaded images to the long side of 512 by keeping original ratio and generate train list.
2. Modify line 14 in main.py  net-> segnet
3. Modify args('dataDir') of train.sh and run train.sh

After training, I found:
1. Inference time on CPU is 3-4 times of your provided weights on (test on both Windows and Mac, PyTorch0.4/0.41/1.0.1). After adding the "torch.set_flush_denormal(True)", speed performs normally, but I don't know what caused this difference.
2. Accuracy is not as good as your provided weights.

I'm wondering if you had ever found these phenomenons. 
It would be great if you could share more details and suggestions. 

Thanks.

**Comment by lizhengwei1992** [comment]
@nulli9 Do you set input size to  256x256 ?  my model trained with about 70K images(coco, supervisely dataset, ..)

**Comment by nulli9** [comment]
@lizhengwei1992 Thanks for your reply.   

1. Yes, input size in both training and inference is set to 256x256. 

2. I also trained the model with coco/supervisely/aiseg/MHP datasets jointly, still obtaining worse performance than yours,  so could you please share the data_list file in your training stage ? 

3.The data preparation process seems to be important, how you processed datasets like supervisely with large height or width?

**Comment by tobechao** [comment]
@nulli9 Hi, Is your problem solved?  I use aifenge dataset, train the t-net poorly
